spec/acceptance/appraisals_file_bundler_dsl_compatibility_spec.rb

```diff
@@ -125,6 +133,10 @@
       source "https://other-rubygems.org" do
         gem "sausage"
         gem "pancake"
+
+        install_if -> { true } do
+          gem 'ketchup'
+        end
```

Comment:
For testing, I'd add another `describe` (and maybe a context, if we're doing that already) for when it's true and then another when it's false (I've given this less thought than the implementation though).